### PR TITLE
Added Intro Command

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
The Intro command is a new command that guides the users through the environment setup and configuration right after CLI installation.

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
If we haven't already, I suggest we test the npm installation locally on both ma, Linux and Windows.
Let's also test the rpm and debian installation locally if we haven't already.

File: build/npm/v2/init.js [hunk 1]
```diff
@@ -68,6 +68,11 @@ function downloadCli() {
     }
 }
 
+function runCliIntro() {
+    const { spawn } = require('child_process');
+    spawn("jf", ['intro'], {stdio: 'inherit', shell: true});
```
[the remark this answers is not in the document]
Comment (omerzi) [reply build/npm/v2/init.js:73]: I found that there is a function "getFileName" that returns the relevant cli ('jfrog'/'jf'), so I switched to that and it works locally. 
Regarding the duplication - I noticed that almost every function is duplicated between v2-jf/init.js and v2/init.js. Is there a reason for this? Would it be better to open a new 'utils.js' file with each duplicated function and call them from each init.js?

File: build/setupcli/jf.sh [hunk 2]
```diff
@@ -69,6 +69,7 @@ while [ -n "$1" ]; do
         then
             echo ""
             echo "The $FILE_NAME executable was installed in $1"
+            echo -e "\e[1;32m Thank you for installing JFrog CLI! 🐸\033[0mֿ"
```
[the remark this answers is not in the document]
Comment (omerzi) [reply build/setupcli/jf.sh:72]: Done

File: main.go [hunk 3]
```diff
@@ -271,3 +281,21 @@ func SetupCmd(c *cli.Context) error {
...
+	ci, err := clientutils.GetBoolEnvValue(coreutils.CI, false)
+	if ci || err != nil {
+		return err
+	}
+	clientLog.Output()
+	clientLog.Output()
+	clientLog.Output(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
+	var serverExists bool
+	serverExists, err = coreConfig.IsServerConfExists()
+	if !serverExists || err != nil {
```
[the remark this answers is not in the document]
Comment (omerzi) [reply main.go]: Done

File: build/npm/v2/init.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (eyalbe4) [reply build/npm/v2/init.js]: I think we should remove all duplications @omerzi. Having duplicate code makes it harder to maintain the code. Please go ahead and remove older duplications as well.

[the remark this answers is not in the document]
Comment (omerzi) [reply build/npm/v2/init.js]: Hey Eyal, 
I tested this and tried to understand how it should be done. There are two separate NPM packages (v2, v2-jf), and I want them to share logic from a third file. There's a problem where they won't recognize the 3rd file unless I make him into a separate npm package and then rely on it as a dependency. However, it seems a bit overkill? We could also copy it when building, but then we won't be able to debug it or see the inner implementations of some functions (the init.js files will call functions that are unknown to them until we build them). Right now as I see it,  the third option is to keep the state as it is with the code duplication, which I guess maybe it is not that bad as this file is not often updated. What do you think will be the proper way to handle this case?

**Pull request merged**